notebooks/working/6.0-mab-estimate-causal-effect.md

```diff
@@ -1841,32 +1841,26 @@ initial_probabilities = mnl_model_sim.predict(long_sim_data)
 
 ```python
 long_sim_data_naive = copy.deepcopy(long_sim_data)
-```
-
-```python
-long_sim_data_naive['total_travel_distance'] = 0.95 * long_sim_data_naive['total_travel_distance']
+long_sim_data_naive['total_travel_distance'] = 1.3 * long_sim_data_naive['total_travel_distance']
 ```
 
 ### 4.2.2. Compute Naive Probabilities Using Disturbed data and Estimated Model on Simulated Data
 
 ```python
 naive_probabilities = mnl_model_sim.predict(long_sim_data_naive)
-```
-
-```python
 long_sim_data_naive['naive_probabilities'] = naive_probabilities
 ```
 
-## 4.2. Based on Causal Graph and Estimated Model from Simulated
+## 4.3. Based on Causal Graph and Estimated Model from Simulated
 
 ```python
 long_sim_data_causal = copy.deepcopy(long_sim_data)
 ```
 
-### 4.2.1. Perturb X and children nodes
+### 4.3.1. Perturb X and children nodes
 
 ```python
-long_sim_data_causal['total_travel_distance'] = 0.95 * long_sim_data['total_travel_distance']
+long_sim_data_causal['total_travel_distance'] = 1.3 * long_sim_data['total_travel_distance']
```

Comment:
Why this particular perturbation?

I'm just thinking that the initial interventions that we spoke of (see #5) all relate to reducing travel distance. Either literally (move people closer to their job) or effectively (via electric bikes as they reduce the effective distance for cycling).

Reply:
@timothyb0912 This was just an example I forgot to clean before I moved on to the loop. Just ignore it. Check out the plots at the bottom. For now the loop only looks at the change in drive alone. I am running it on my laptop right now to plot the estimated effects on all driving modes.

Reply:
Ah, I see later that you use a perturbation 0.8.

This is cool. You show that both ways (increasing or decreasing travel distance) leads to incorrect causal effect inferences.

👍